localgov_skeleton.info.yml

```diff
@@ -5,6 +5,9 @@ core_version_requirement: ^8.8 || ^9
 
 base theme: classy
 
+libraries:
+  - localgov_skeleton/global
```

Comment:
This means all subthemes of localgov_skeleton will include the [assets/css/gds.css](https://github.com/localgovdrupal/localgov_skeleton/blob/master/assets/css/gds.css) file in each and every page.  Do we really want that?

FAO @cjstevens78 @finnlewis @ekes

Reply:
Good question!

Personally, I think this is fine. This is just the form elements for webforms, and we are trying to encourage gov.uk best practices where possible.

A subtheme could remove this if needed. 

@markconroy any thoughts on this?

Reply:
If I get my way, we'll be loading the whole of the govuk frontend in all our themes so that we can leverage from it's CSS and JS, so I don't see any issue  that small gds.css file being added. It's base CSS and not specific to any component - will be used on exposed filters, facets, webforms, etc, so should be added globally in my opinion.

Reply:
Thanks @markconroy 

@Adnan-cds any objections to merging this and tagging the release?

Reply:
> any objections to merging this and tagging the release?

Okay, go ahead with this one if it only effects Webforms.

> we'll be loading the whole of the govuk frontend in all our themes

I am not too keen on embracing the govuk frontend though.  The Croydon site's UX design doesn't aspire to follow the govuk frontend as far as I understand.  Correct me if I am wrong Dave (@clearspotmedia).

Also, localgov_theme is an existing theme, so if you bend it too much, it will break existing sites.  My suggestion would be to implement the govuk frontend on a child theme of localgov_skeleton.

Reply:
Be interesting to see how localgov drupal looks with the [gov_uk theme](https://www.drupal.org/project/govuk_theme)

If we are going to make substantial theme changes such as adopting the gov.uk design system then thats something that will need to go via product group and any councils building sites using the current localgov_theme which is the design work from BHCC / Croydon. So changes will be like the field renaming release.

Maybe there is scope to do a GDS design system flavour of the localgov_theme, or even have a module that brings in the GDS design system like the bootstrap / font awesome module?

Reply:
Looking at the code for the gov_uk theme linked above, it's very specifically for a site that would be hosted on the govuk network (as in a GovUK service or something, rather than an individual site like a council's site).

I wouldn't propose using that for LocalGovDrupal. What I'd propose is we add the library so we can then utilise the classes we want for trickier stuff like tabs, accordions, etc. And individual themes can leverage more by simply adding CSS classes to their templates if they wish.

Reply:
Discussed today in the standup, no objections to merging this.

Reply:
So to be clear the CSS in master is going into the release, and it is largely for GDS type form elements, implemented specifically within the skeleton.

The second discussion on this thread is looking at the gov_uk theme, which should at least contain some of these GDS styled components. Going forward there will be a new theme might want to leverage some of that, but maybe not actually inherit it.